chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SearchViewController.java

```diff
@@ -393,4 +400,20 @@ public void download(@RequestParam(value="edition", required=false) String editi
 		return searchMenuManager.getPopulateSearchOptions(simple);
 	}
 	
+	@ApiOperation(value="Get all developer decertifications in the CHPL", 

```

Comment:
The title of OCD-1016 in Jira is "Add new /search for decertified Developers". Given the title name, I figured that if I am going to add a search, then I would do it in the SearchViewController. Should I move the code to the DeveloperController or a DecertificationsController?